### DISCUSSION TITLE
WSL Install Help

### DISCUSSION THREAD

**Discussion by malydilnar** [OpenRadioss/OpenRadioss#1143] [Q&A] [answered]
Hey all, 

Working on getting OpenRadioss bult in WSL but keep running into issue that running **ldd starter_linux_gf** shows that 

**libhm_reader_linux64.so => not found**

Ive tried to follow various sources on the internet on how to correct this but no luck so far. I've tried setting and saving the path variable in the bashrc file but no luck.I've been referencing the following: files or pages trying to get this working.

install.md 
https://github.com/OpenRadioss/OpenRadioss/discussions/419?sort=new
https://www.reddit.com/user/welsim/comments/xb5f9q/explore_the_enterprisegrade_opensource_solver/

My bashrc:
![help1](https://user-images.githubusercontent.com/60806917/220000009-3686d618-1a5f-45fa-a9f4-9bca2d91dc8a.PNG)

after running **ldd starter_linux_gf**
![help2](https://user-images.githubusercontent.com/60806917/220000017-0eba0a66-eaeb-4235-967b-d3fadfd6ea34.PNG)

I've use OpenFoam a bunch but not a knowledgeable Linux user. Any help would be much appreciated. Thanks

**Comment by laurent-altr** [comment, marked as answer]
Hi,

From your second snapshot, it seems that OpenRadioss directory is  ~/OpenRadioss, but you set OPENRADIOSS_PATH to /OpenRadioss (instead of ~/OpenRadioss) in your .bashrc file.

**Reply by malydilnar** [reply]
This fixed it! Thank you so much
